x-pack/plugins/uptime/public/lib/helper/component_test_helpers.tsx

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
i think file needs to be removed.

Reply:
yeah it was my mistake in a previous merge, I went in and got rid of it.